packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts

```diff
@@ -44,14 +44,18 @@ export class CloudinaryMediaStore implements MediaStore {
       throw new Error(responseData.message)
     }
     const fileRes = await res.json()
+    /**
+     * Format the response from Cloudinary to match Media interface
+     */
+    const parsedRes: Media = {
+      type: fileRes.resource_type === 'image' ? 'file' : 'dir',
+      id: fileRes.public_id,
+      filename: fileRes.original_filename,
+      directory: '/',
+      previewSrc: fileRes.url
 
-    // TODO: be programmer
-    // NOTE: why do we need this?
```

Comment:
just noting here, I believe images were taking a second to appear in the cloudinary API response after being uploaded, so this delay was just a bodge that waits a couple seconds before resolving `persist` to help ensure the new image is there. If removing this doesn't seem to be causing any problems then it's probably fine, but I'd keep an eye out for it

Reply:
note that the scenario in which this problem will most likely appear is when uploading an image to the media manager from the media manager UI. In that case it does a re-fetch after the upload, and if the re-fetch happens too soon, the new image will not appear.

Reply:
Ah, I was definitely running into that. I'll keep testing it out to check for consistency, but at the very least we can bring that back.